bigchaindb/commands/bigchaindb.py

```diff
@@ -143,6 +144,36 @@ def run_upsert_validator_new(args, bigchain):
         raise OperationError('Failed to commit election')
 
 
+def run_upsert_validator_approve(args, bigchain):
+    """Approve an election to add/update/remove a validator to an existing BigchainDB network
+
+    :param args: dict
+        args = {
+        'election_id': the election_id of the election (str)
+        'sk': the path to the private key of the signer (str)
+        }
+    :param bigchain: an instance of BigchainDB
+    :return: a success message
+    :raises: OperationError if the write transaction fails for any reason
+    """
+
+    key = load_node_key(args.sk)
+    tx = bigchain.get_transaction(args.election_id)
+    voting_power = [v.amount for v in tx.outputs if key.public_key in v.public_keys][0]
```

Comment:
In case there are no votes allocated to the public key, the above code raises an exception. It would be better to catch the exception and inform the user that they don't have the right to vote on this election.